docs/tutorial/getting-started.md

```diff
@@ -0,0 +1,78 @@
+# Getting started
+
+In this tutorial, you will set up a local version of our WordPress network.
+
+## Prerequisites
+
+Please ensure you have the following tools set up on your computer. Everything
+can be installed via [Homebrew](https://brew.sh/).
+
+### PHP
+
+We currently use the latest release on the 8.0.x branch, using commands such as:
+
+```bash
+$ brew install php@8.0
```

Comment:
```
php --version                                                                                                    15:31:19
PHP 8.2.4 (cli) (built: Mar 16 2023 16:25:32) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.2.4, Copyright (c) Zend Technologies
    with Zend OPcache v8.2.4, Copyright (c), by Zend Technologies
```
uh... oops

Reply:
Hopefully that bodes well for when we get around to upgrading PHP versions :-)